### ISSUE TITLE
Two to Three mask word prediction at the same sentence is very complex

### ISSUE THREAD

**Issue by MuruganR96** [huggingface/pytorch-pretrained-BERT#204]
Two to Three mask word prediction at the same sentence also very complex.
how to get good accuracy?
if i have to pretrained bert model and own dataset with **masked_lm_prob=0.25** (https://github.com/google-research/bert#pre-training-with-bert),  what will happened?

Thanks.

**Comment by thomwolf** [comment]
Hi @MuruganR96, from my experiments two to three mask word prediction doesn't seems to work with BERT.

**Comment by MuruganR96** [comment]
thanks @thomwolf  sir
